addons/print/views/account_invoice_view.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <!-- Add entry action in the 'More' Menu -->
+        <record id="action_print_account_invoice_multi" model="ir.actions.act_window">
+            <field name="name">Send Invoices by post</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">print.order.wizard</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">form</field>
+            <field name="view_id" ref="print_order_wizard_form_view"/>
+            <field name="target">new</field>
+            <field name="multi">True</field>
+        </record>
+
+        <record id="print_account_invoice_multi" model="ir.values">
+            <field name="model_id" ref="account.model_account_invoice" />
+            <field name="name">Send invoices by post</field>
+            <field name="key2">client_action_multi</field>
+            <field name="value" eval="'ir.actions.act_window,' + str(ref('action_print_account_invoice_multi'))" />
+            <field name="key">action</field>
+            <field name="model">account.invoice</field>
+        </record>
+
+
+        <!-- Add button on invoice form view -->
+        <record id="action_print_invoice_wizard" model="ir.actions.act_window">
```

Comment:
So, according to the new naming spec, it should be print_order_wizard_action_print ? If I understood everything, all ids should be updated :p .